plugin/core/protocol.py

```diff
@@ -137,6 +137,10 @@ def rename(cls, params: dict) -> 'Request':
     def codeAction(cls, params: dict) -> 'Request':
         return Request("textDocument/codeAction", params)
 
+    @classmethod
+    def color(cls, params: dict) -> 'Request':
```

Comment:
I'm a bit divided about the naming - would prefer 'documentColor' but obviously we're not totally consistent here.

Reply:
I will put documentColor. It will be more consistent that way